sbt-scalajs-bundler/src/main/scala/scalajsbundler/BundlerFile.scala

```diff
@@ -54,20 +56,32 @@ object BundlerFile {
     */
   case class WebpackConfig(application: Application, file: java.io.File)
       extends Internal {
-    def project = application.project
+    def project: String = application.project
 
     def targetDir: File = file.getParentFile
 
-    def asLibrary: Library =
+    def asLibrary(stats: Option[WebpackStats]): Library =
       Library(project,
               file.getParentFile.toPath
-                .resolve(Library.fileName(project))
+                .resolve(stats.flatMap(_.assetName(project)).fold(Library.fileName(project))(identity))
```

Comment:
I’m wondering if we should make the `asLibrary` method return an `Option[Library]` instead. Is there a case where `assetName(project)` will fail but `Library.fileName(project)` will be *correct*?

But maybe such a change will have too many consequences and should be addressed in another PR…

Reply:
Thinking again about this, I think we should return *all* the bundles produced by webpack, not just the one that has the same name as the project. What do you think?

Reply:
I want to export all the bundles. I'd use them for packaging, so I can define assets in webpack bun package e.g. with `sbt-native-packager`. i was thinking to do that in another PR though

Reply:
I've been playing a bit with this and to make a clean solution it will require some changes to the `BundleFile` hierarchy. It currently assumes it is a single file but we want to make it more flexible. I don't know if it is better to do that in another PR